src/test/java/api/InstancesApiExamples.java

```diff
@@ -308,6 +308,17 @@ public void shouldReturnServerErrorIfOneOfInstancesPostedWithoutTitle() throws M
     assertEquals(postResponse.getJson().getInteger("totalRecords"), Integer.valueOf(2));
   }
 
+  @Test
+  public void shouldReturnBlockedFieldsConfig() throws MalformedURLException, InterruptedException, ExecutionException, TimeoutException {
+    CompletableFuture<Response> getCompleted = new CompletableFuture<>();
```

Comment:
Personally, I find it easier to read code when variable declaration and usage are separated by whitespace.

Reply:
Agree